src/main/java/com/iexec/sms/secret/MeasuredSecretService.java

```diff
@@ -52,18 +53,21 @@ public class MeasuredSecretService {
     private Counter initialSecretsCounter;
     private AtomicLong storedSecretsCount;
     private final Counter addedSecretsSinceStartCounter;
+    private AtomicLong cachedSecretsCount;
```

Comment:
What do you think of changing all `AtomicLong` as `final` in constructor and only do a set of `storedSecretsCount` in `@PostContruct` ?